### PR TITLE
Use DelegateContribute votingPower for Tweet Proposal

### PR DESCRIPTION
**Pull request by truemiller**
https://autonolas-contribute-frontend-git-tweet-delegation-autonolas.vercel.app/tweet

Implements new DelegateContribute contract, and checks for "votingPower" instead of measuring votes via the veOLAS contract. This lets EOAs vote on tweet proposals if they have been delegated to.

Staging allows users to propose without 100K voting power, production will enforce the a 100K voting power check.

* May need to switch chain to Ethereum on staging, as we also support Goerli on staging. The contract is only Ethereum mainnet at present.

### CODE REVIEW

**Review by oaksprout — DISMISSED** [review]
working now – haven't reviewed code though

File: common-util/Login/config.jsx [hunk 1]
```diff
@@ -14,7 +14,7 @@ import { RPC_URLS } from 'common-util/Contracts';
 export const SUPPORTED_CHAINS = (
   process.env.NEXT_PUBLIC_PFP_URL || ''
 ).includes('staging')
-  ? [goerli]
+  ? [goerli, mainnet]
```
[the remark this answers is not in the document]
Comment (truemiller) [reply common-util/Login/config.jsx:17]: yes, the DelegateContribute contract is only on mainnet currently

File: common-util/AbiAndAddresses/delegateContributeContract.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+  inputs: [{ internalType: 'address', name: 'delegatee', type: 'address' }], name: 'delegate', outputs: [], stateMutability: 'nonpayable', type: 'function',
+}, {
+  inputs: [{ internalType: 'address', name: 'delegatee', type: 'address' }], name: 'getDelegatorList', outputs: [{ internalType: 'address[]', name: '', type: 'address[]' }], stateMutability: 'view', type: 'function',
+}, {
+  inputs: [{ internalType: 'address', name: '', type: 'address' }], name: 'mapDelegation', outputs: [{ internalType: 'address', name: '', type: 'address' }], stateMutability: 'view', type: 'function',
+}, {
+  inputs: [], name: 'veOLAS', outputs: [{ internalType: 'address', name: '', type: 'address' }], stateMutability: 'view', type: 'function',
+}, {
+  inputs: [{ internalType: 'address', name: 'delegatee', type: 'address' }], name: 'votingPower', outputs: [{ internalType: 'uint256', name: 'totalVotingPower', type: 'uint256' }], stateMutability: 'view', type: 'function',
+}];
```
[the remark this answers is not in the document]
Comment (truemiller) [reply common-util/AbiAndAddresses/delegateContributeContract.js:22]: tested, this is the format suggested :/

File: components/MembersList/requests.jsx [hunk 3]
```diff
@@ -10,9 +10,22 @@ export const fetchVeolasBalance = async ({ account }) => {
...
 export const checkVeolasThreshold = async (account, thresholdInWei) => {
   const balance = await fetchVeolasBalance({ account });
   const bNBalance = BigNumber.from(balance);
   const processedThreshold = BigNumber.from(thresholdInWei);
   return bNBalance.gte(processedThreshold);
 };
+
+export const checkVotingPower = async (account, thresholdInWei) => {
+  const votingPower = await fetchVotingPower({ account });
+  const bNBalance = BigNumber.from(votingPower);
```
Comment (Tanya-atatakai) [inline components/MembersList/requests.jsx:28]: tiny, I guess copypasted - should be bNVotingPower

**Review by Tanya-atatakai — APPROVED** [review]

File: components/MembersList/requests.jsx (hunk 3 above)
Comment (truemiller) [reply components/MembersList/requests.jsx]: done!

**Review by Tanya-atatakai — APPROVED** [review]

**Review by kupermind — APPROVED** [review]

**Review by oaksprout — APPROVED** [review]

**Review by DavidMinarsch — APPROVED** [review]

**Pull request merged**